src/config.ts

```diff
@@ -1,8 +1,9 @@
 const config = {
   REDIRECT_URI: 'REPLACE_WITH_YOUR_REDIRECT_URI', // i.e http://localhost:3000, https://serverName/lf-sample/index.html
+  REDIRECT_URI_POPUP: 'REPLACE_WITH_YOUR_REDIRECT_URI', // i.e http://localhost:3000/static-assets/login.html, https://serverName/lf-sample/static-assets/login.html 
```

Comment:
should we instead just add login.html to the end of the redirect_uri

to that end, should we not even make them set a redirect_url and just use the current location

Reply:
I think that might be simpler and less confusing
Especially since they will be using the same reference location and only their serverName/local host port might change